### DISCUSSION TITLE
How to load Draco glb offline

### DISCUSSION THREAD

**Discussion by FEJackFly** [visgl/deck.gl#8630] [Q&A]
https://unpkg.com/@loaders.gl/draco@3.4.4/dist/draco-worker.js   Failure to load

**Comment by Pessimistress** [comment]
https://deck.gl/docs/developer-guide/loading-data#loaders-and-web-workers

**Reply by FEJackFly** [reply]
<img width="523" alt="image" src="https://github.com/visgl/deck.gl/assets/24688308/2591883a-cec8-4e73-ad16-db030a317c91">
<img width="975" alt="image" src="https://github.com/visgl/deck.gl/assets/24688308/abfacb00-65d9-4ba0-b162-3c21e83298f7">
When I disabled worker, there were two more network requests that prevented offline use

**Reply by ibgreen** [reply]
Yes DracoLoader by default loads the big Draco libraries from CDN but you may be able to install and import draco yourself and pass it in as {..., modules: {draco3d: ...}} in the loader options. It is a rare use case and not heavily tested or documented but if you can put breakpoints in the draco loader that may let you see what is going on.
